app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -53,6 +56,7 @@
     private static final String TAG = "SearchFragment";
     private static final String ARG_QUERY = "query";
     private static final String ARG_FEED = "feed";
+    private static String FEED_TITLE = "";
```

Comment:
Please rename to `private String feedTitle` (and move a bit down). Static members that store actual data can cause issues when using the class multiple times.

Reply:
Understood, the problem would be that I cant reference non static members from the newInstance static method, how should I proceed?

Reply:
I was able to do It with this line:
`fragment.feedTitle = feedTitle;`

is this the correct approach?

Reply:
> I was able to do It with this line: fragment.feedTitle = feedTitle;

This is a problem when the Android system recreates the fragment after the app was in background

Have a look at the comment below. You can add it as an argument (`fragment.getArguments().putString(ARG_FEED_NAME, name);`).

Reply:
Ohhhh, know I understand... I am about to push all the changes now :)